content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +83,95 @@ and has not been altered during transport.
 
 How to create or modify a CNAME record depends on the provider hosting the DNS zone of your domain
-name. The most common providers are list below.
+name. The most common providers are
+:ref:`listed below <email_communication/SPFDKIM_common_providers>`.
 
-You can check if your DKIM record is valid with a free tool like 
-`DKIM Core <https://dkimcore.org/tools/>`_. If a selector is asked, enter `odoo`.
+You can check if your DKIM record is valid with a free tool like `DKIM Core <https://dkimcore.org
+/tools/>`_. If a selector is asked, enter `odoo`.
 
 Check your DMARC policy
 =======================
 
-The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies SPF
-and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
+The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies
+SPF and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
 server what to do with an incoming email that fails the SPF and/or DKIM check.
 
 There are three DMARC policies:
-- ``p=none``
-- ``p=quarantine``
-- ``p=reject``
 
-``p=quarantine`` and ``p=reject`` instruct the server that receives an email to quarantine that
-email or ignore it if the SPF and/or DKIM check fails.
+- `p=none`
+- `p=quarantine`
+- `p=reject`
 
-If your domain name uses DMARC and has defined one of these policies, it is therefore imperative 
-to be SPF compliant or to enable DKIM.
+`p=quarantine` and `p=reject` instruct the server that receives an email to quarantine that email
+or ignore it if the SPF and/or DKIM check fails.
+
+If your domain name uses DMARC and has defined one of these policies, it is therefore imperative to
+be SPF compliant or to enable DKIM.
 
 .. danger::
-   Yahoo or AOL are examples of email providers with a DMARC policy set to ``p=reject``. We
-   strongly advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails
-   will never reach their recipient.
+   Yahoo or AOL are examples of email providers with a DMARC policy set to `p=reject`. We strongly
+   advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails will
+   never reach their recipient.
 
-``p=none`` is used for the domain owner to receive reports about entities using their domain. It 
+`p=none` is used for the domain owner to receive reports about entities using their domain. It
 should not impact the deliverability if the DMARC check fails.
 
-You can check the DMARC record of a domain name with a tool like
-`MXToolbox DMARC <https://mxtoolbox.com/DMARC.aspx>`_.
+You can check the DMARC record of a domain name with a tool like `MXToolbox DMARC <https://mxtoolbo
+x.com/DMARC.aspx>`_.
 
 If one of your partners, customer or vendor, uses DMARC and has defined one of these policies, the
 Odoo server cannot relay emails from this partner to your users.
```

Comment:
- Replace `one of your partners` with `a partner`
- Add a comma after `customer`
- Replace `your users` with `the database users`